BunnymodXT fails to compile for windows

I just cloned the repo (commit I'm using is https://github.com/YaLTeR/BunnymodXT/commit/56bb64c35190f79177b739b9b4abfc594a6122cb) and tried to build it on windows. I got the following error:

```
BunnymodXT\BunnymodXT\helper_functions.hpp(27,4): error C3861: 'EngineMsg': identifier not found
```

I then looked through the commit/ci history on github and saw that ci has been failing for windows builds for the exact same reason:
https://github.com/YaLTeR/BunnymodXT/actions/runs/10122561202/job/27994847466#step:7:127

This error started happening with this commit:
https://github.com/YaLTeR/BunnymodXT/commit/0e208181e3359939b580a9aaf951594245bc42fb

--- a/BunnymodXT/helper_functions.hpp
+++ b/BunnymodXT/helper_functions.hpp
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "sptlib-wrapper.hpp"
+
 #ifdef _WIN32
 	#define DLL_EXTENSION ".dll"
 	#define PATH_SLASH '\\'

